db/migrate/20190423095925_add_names_to_allocation.rb

```diff
@@ -0,0 +1,11 @@
+class AddNamesToAllocation < ActiveRecord::Migration[5.2]
+  def up
+    add_column :allocations, :pom_name, :text
+    add_column :allocations, :created_by_name, :text
```

Comment:
Is it worth amending the existing 'created_by' column to 'created_by_username' so it's the same level of granularity?

Reply:
Yes, that's not a bad idea.